servicedirectory/namespaces_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 Google LLC
```

Comment:
Looks like namespaces are tested by endpoints_test.go. I think we can delete this. It's OK to have all of the samples tested together if they are all interconnected.